### DISCUSSION TITLE
Links broken in docs for tutorial

### DISCUSSION THREAD

**Discussion by technophile-04** [SimpleFi-finance/subgraphs#58] [Q&A] [answered]
Hii I was following along the tutorial: https://docs.simplefi.finance/subgraph-development-documentation/sushiswap-subgraph-development/start-subgraph-development

It has broken link :  https://github.com/SimpleFi-finance/subgraphs/blob/master/simplefi-sushiswap/subgraph.yaml

Could someone please help me find this `simplefi-sushiswap` directory 🙏

**Comment by gvladika** [comment, marked as answer]
Hi @technophile-04 !

Good catch, we have merged codebases of Sushiswap subgraph with the Uniswap subgraph, as Sushiswap smart contracts are fork of Uniswap. That's the reason why we use single codebase for every Uniswap-fork subgraph. You can find it here:
https://github.com/SimpleFi-finance/subgraphs/tree/master/uniswap-v2

We will update linkin docs, let us know if you have any other questions in the meantime!

**Reply by technophile-04** [reply]
Thanks so much @gvladika 🙏...Also thanks to the SimpleFi team for creating such a great tutorial! Learning a lot from it 🙏🙏
